src/System.Net.Primitives/tests/FunctionalTests/Cookie.cs

```diff
@@ -0,0 +1,344 @@
+﻿using System;
+using System.Net;
+
+using Xunit;
+
+namespace System.Net.Primitives.Functional.Tests
+{
+    public static class CookieTest
+    {
+        [Fact]
+        public static void TestConstructors()
```

Comment:
Would you mind splitting each of these cases out into its own `[Fact]` for better pass/fail granularity (and similarly for the other tests throughout this change)?